### DISCUSSION TITLE
more testing?

### DISCUSSION THREAD

**Discussion by wonder99** [springbok/MLM2PRO-GSPro-Connector#34] [General]
Hi,
sorry I've been focusing on features for the other connector.  But that's done now, so I have time for testing this, if you want.  Is there an area you'd like me to test?  I have android phone, tablet, and iphone6s.

**Comment by springbok** [comment]
No worries at all. I've had my head down developing the GUI version which I released this morning. Would be great if you could give it a go. Also will be looking at integrating putting so any help on that side would be greatly appreciated.

**Reply by springbok** [reply]
Hi Paul,

I’ve got a pre-release with putting if you are interested in testing it. It uses some of the great work you did trailblazing the putting integration, will definitely acknowledge your great work in the next release.

**Reply by springbok** [reply]
Just out of interest I did start grabbing screenshots directly from the capture card thinking that might be the better and simpler way but quickly realized having the camera app and a view of the exputt screen was invaluable, and there was no point reinventing the wheel by writing an equivalent.

**Comment by wonder99** [comment]
Ya, the Camera app is handy.  I also find it handy to have the
tempo/metronome option enabled, so that you can hear when Exputt is ready
for the shot.
I'm on vacation for the next few days, but will test when I get back.
- Paul

**Comment by wonder99** [comment]
I've just gotten back to testing.  V1.00.00 looks good, but I can't find a
pre-release with putting.  Where is it?
- Paul

**Comment by wonder99** [comment]
I did get this message, after a little testing--couldn't access the log
file.  I found 4 copies of the connector still running.
[image: image.png]
- Paul

**Comment by wonder99** [comment]
And is the connector supposed to be able to launch GSPRO?
[image: image.png]
[image: image.png]

**Comment by wonder99** [comment]
One more thing--when I use the shortcut .lnk file to launch my screen
mirror, it takes too long and the connector says it can't find the window.
When I try to connect again, it does find it.
I think we need a little delay before declaring it can't find the screen
mirror app.
- Paul

**Comment by wonder99** [comment]
Another one.  I caught it reporting 2 shots for my one shot.  see the image
below
[image: image.png]

I think the repeat shot was because it measured a difference in spin, so
thought it was a new shot.  I recommend if spin > 15000, divide it by 10
until it's under 15k.  Report that, and when looking for a change in the
screen mirror results, only compare the post-divided values.
That's what I did in the other connector.
- Paul

**Reply by springbok** [reply]
Hi Paul I appreciate the testing and feedback I’ll have a look at these issues this weekend, I’ve been in hospital last couple of days hopefully get out today. Here is the pre-release with putting included:

https://1drv.ms/u/s!Agx7aYfjrZJ8nBiwQTilVjRK6OX7?e=thFpKp

**Reply by springbok** [reply]
The only change since this release has been removal of the capture card ID as I don’t use it.

**Comment by wonder99** [comment]
I just did a bunch of testing tonight with ExPutt.  No real issues to
report!  great!

- Paul
